README.md

```diff
@@ -1,707 +1,1799 @@
+| `resultFunc`                    | The final function passed to [`createSelector`]. Otherwise known as the `combiner`.                                                                                                               |
+| `memoizedResultFunc`            | The memoized version of `resultFunc`.                                                                                                                                                             |
+| `lastResult`                    | Returns The last result calculated by `memoizedResultFunc`.                                                                                                                                       |
+| `dependencies`                  | The array of the input selectors used by [`createSelector`] to compose the combiner (`memoizedResultFunc`).                                                                                       |
+| `recomputations`                | Counts the number of times `memoizedResultFunc` has been recalculated.                                                                                                                            |
+| `resetRecomputations`           | Resets the count of `recomputations` count to 0.                                                                                                                                                  |
+| `dependencyRecomputations`      | Counts the number of times the [`input selectors`] ([`dependencies`]) have been recalculated. This is distinct from `recomputations`, which tracks the recalculations of the [`result function`]. |
+| `resetDependencyRecomputations` | Resets the `dependencyRecomputations` count to 0.                                                                                                                                                 |
+| `memoize`                       | Function used to memoize the `resultFunc`.                                                                                                                                                        |
+| `argsMemoize`                   | Function used to memoize the arguments passed into the [`output selector`].                                                                                                                       |
+
+</details>
+
+<div align="right">[ <a href="#table-of-contents">↑ Back to top ↑</a> ]</div>
+
+---
+
+<a id="v5summary"></a>
+
+## What's New in 5.0.0?
+
+<details><summary><b>Version 5.0.0 introduces several new features and improvements:</b></summary>
+
+- **Customization Enhancements**:
+
+  - Added the ability to pass an options object to [`createSelectorCreator`], allowing for customized `memoize` and `argsMemoize` functions, alongside their respective options (`memoizeOptions` and `argsMemoizeOptions`).
+  - The [`createSelector`] function now supports direct customization of `memoize` and `argsMemoize` within its options object.
```

Comment:
:hand: I'd put this one above the `createSelectorCreator` customization, really :)

Reply:
You wish is my command :)